src/anemoi/training/data/datamodule.py

```diff
@@ -185,7 +189,7 @@ def _get_dataloader(self, ds: NativeGridDataset, stage: str) -> DataLoader:
             num_workers=self.config.dataloader.num_workers[stage],
             # use of pinned memory can speed up CPU-to-GPU data transfers
             # see https://pytorch.org/docs/stable/notes/cuda.html#cuda-memory-pinning
-            pin_memory=True,
+            pin_memory=self.config.dataloader.pin_memory,
```

Comment:
See above comment about using `.get`

Reply:
Thanks for your quick review. I've implemented you change requests.